backend/backend/middleware/profile.py

```diff
@@ -21,9 +21,9 @@ def __call__(self, request):
             response = self.get_response(request)
 
             profiler.disable()
-            stamp = "%s" % (request.path.replace("/", "__"),)
-            profiler.dump_stats('/tmp/%s.pro' % stamp)
-            print("Dumped profile info in /tmp/%s.pro" % stamp)
+            stamp = f'{request.path.replace("/", "__")}'
```

Comment:
We can use the string directly, don't need interpolation

Reply:
I wondered, but just noted it to come back to after doing all the conversions. I've now corrected it to:

```
stamp = request.path.replace("/", "__")
```